packages/apps/marmalade-marketplace/src/components/GuardForm/index.tsx

```diff
@@ -2,9 +2,8 @@ import React, { FC } from 'react';
 import { TextField, Checkbox } from '@kadena/kode-ui';
 import CrudCard from '@/components/CrudCard';
 
-
 interface GuardFormProps {
-  guardInput: { [key: string]: string };
+  guardInput: { [key: string]: any };
```

Comment:
The sdk has exposed a `Guard` type you could use here.

Reply:
nvm, I thought it was a guard type

Reply:
I think `Guard` is declared locally and not exported